# Measuring the robustness of microbiome-disease associations

For access to data used in this analysis (e.g. the T2D_90.rds file used in the revision analysis), see our Figshare repository: https://figshare.com/projects/Microbiome_robustness/127607

# Manuscript abstract

Associating human gut microbes with disease is one prerequisite to determining causal roles of the microbiome in human health. We assessed the analytical robustness of taxonomic associations across 6 prevalent and well-studied diseases using 15 public cohorts (2,343 samples). We executed a large sensitivity analysis to find consistent disease indicators across millions of models, focusing specifically on 581 previously published results. Systematically exploring combinations of model specifications revealed associations that could potentially be overlooked when restricting analyses to one or a few modeling strategies. Additionally, we found that at least 20% of association directions for a given feature conflicted (could be positive or negative depending on model specification) in one third  of all 581 taxa tested. Published associations in type 1 and type 2 diabetes (e.g. the relationship between Streptococcus and type 1 diabetes) were particularly non-robust, with our pipeline failing to re-identify concordant results for >90% of reported findings. Further, we quantified the influence of variables like sequencing depth, glucose levels, cholesterol, and age while re-identifying the prominence of confounders such as medication (e.g. metformin) and body mass index (BMI). We focus specifically on how these variables influence reported associations with a well-studied bacteria reported to be correlated negatively with disease, F. prausnitzii. Overall, we have identified drivers of inconsistency in microbiome models as well as shown how sensitivity analysis can recover or identify new potentially biologically relevant microbiome-disease associations.

# Contact

Braden Tierney
btt4001 ~at~ med DOT cornell DOT edu
Twitter
@BradenTierney
